### ISSUE TITLE
Why is rebar3 included?

### ISSUE THREAD

**Issue by Zalastax** [sile/jsone#42]
It's very strange to me that the rebar3 binary is included in the repo.
Travis already supports rebar3 without including it manually and it's very easy to get hold of as a developer.
Can it be removed?

**Comment by sile** [comment]
You are right. There are only historical reasons to include rebar3 binary in this repository. I will remove it. Thanks!
